Expired/Views/SettingView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SettingView.swift
+//  Expired
+//
+//  Created by Karl Michael Reyes on 2023-02-06.
+//
+
+import SwiftUI
+
+struct SettingView: View {
+    @Environment(\.managedObjectContext) private var viewContext
+    @EnvironmentObject var productStore: ProductStore
+    @State private var showingDeleteAlert: Bool = false
+    @State private var showingSuccessAlert: Bool = false
```

Comment:
```suggestion
    @State private var showingDeleteResultAlert: Bool = false
    @State private var deleteSuccess: Bool = false

```

- Rename `showingDeleteAlert` to `showingDeleteResultAlert`, more details below
- Add `deleteSuccess`, more details below
- Add an empty line between wrapped properties and `body`